include/roboteam_ai/treeinterp/BTFactory.h

```diff
@@ -47,10 +49,17 @@ class BTFactory {
 
         static bool hasMadeTrees();
 
-    private:
+
+
+private:
         static std::string currentTree;
         static std::string keeperTree;
         static bool weMadeTrees;
+
+
+
```

Comment:
I know you need somewhere to breath clean air after reading this code but you won't find it by adding extra space to our code.